docs/topic-guides/kotlin/suggest.md

```diff
@@ -0,0 +1,85 @@
+# Suggest
+
+The API for the `SuggestStore` can be found in the [MozillaComponents Kotlin documentation](https://mozilla.github.io/application-services/kotlin/kotlin-components-docs/mozilla.appservices.suggest/-suggest-store/index.html).
+
+> Make sure you initialize [`viaduct`](../viaduct.md) for this component.
+
+> The `SuggestStore` is a synchronous, which needs to be wrapped in the asynchronous primitive of the target language you are using it in.
+
+## Setting up the store
+
+You need to import one or more of the following primitives to work with the `SuggestStore` (these come from the generated `suggest.kt` file, produced by `uniffi`):
+
+```kotlin
+import mozilla.appservices.remotesettings.RemoteSettingsServer
+import mozilla.appservices.suggest.SuggestApiException
+import mozilla.appservices.suggest.SuggestIngestionConstraints
+import mozilla.appservices.suggest.SuggestStore
+import mozilla.appservices.suggest.SuggestStoreBuilder
+import mozilla.appservices.suggest.Suggestion
+import mozilla.appservices.suggest.SuggestionQuery
+```
+
+Create a `SuggestStore` as a singleton. You do this via the `SuggestStoreBuilder`, which returns a `SuggestStore`.  No I/O or network requests are performed during construction, which makes this safe to do at any point in the application startup:
+
+```kotlin
+internal val store: SuggestStore = {
+    SuggestStoreBuilder()
+        .dataPath(context.getDatabasePath(DATABASE_NAME).absolutePath)
+        .remoteSettingsServer(remoteSettingsServer)
+        .build()
+```
+
+* You need to set the `dataPath`, which is the path (the SQLite location) where you store your suggestions.
+* The `remoteSettingsServer` is only needed if you want to set the server to anything else but `prod`. If so, you pass a `RemoteSettingsServer` object. 
+
+## Ingesting suggestions
+
+We have to handle different phases of the ingesting suggestions: The initial one on starting up the application, and then, periodically, in the background.
+
+* [`SuggestIngestionConstraints`](https://mozilla.github.io/application-services/kotlin/kotlin-components-docs/mozilla.appservices.suggest/-suggest-ingestion-constraints/index.html?query=data%20class%20SuggestIngestionConstraints(var%20providers:%20List%3CSuggestionProvider%3E?%20=%20null,%20var%20providerConstraints:%20SuggestionProviderConstraints?%20=%20null,%20var%20emptyOnly:%20Boolean%20=%20false)) is used to control what gets ingested.
+* Use the `providers` field to limit ingestion by provider type (show an example where you're only ingesting a couple providers. `AMP_MOBILE`, `WIKIPEDIA`, and `WEATHER` are a reasonable combination.
+* Use the `providerConstraints` field to add additional constraints, currently this is only used for exposing suggestions.
```

Comment:
```suggestion
* Use the `providerConstraints` field to add additional constraints, currently this is only used for exposure suggestions.
```

"exposure suggestions" are a new suggestion type that the SnG team is introducing.  They're going to be used to measure if a user would be exposed to a suggestion type if it was introduced with a given set of keywords.  The user actually won't be exposed to them, it's all handled internally.